sonarqube/ocp-config/sonarqube.yml

```diff
@@ -436,6 +436,5 @@ parameters:
 - name: REPO_BASE
   required: true
 - name: STORAGE_CLASS_DATA
-  required: true
 - name: STORAGE_PROVISIONER
-  required: true
+- name: STORAGE_CLASS_BACKUP
```

Comment:
Could you enable trailing newlines in your editor? That would be great. I think we actually wanted to document that somewhere ...

Reply:
Hi, yes, there is an open issue about coding conventions. I have activated it in my editor now.